test/unit/spec/EnrollChoices_spec.js

```diff
@@ -775,6 +815,16 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
               expect(test.form.factorCardinalityText(factorName)).toBe('(3 set up)');
             });
         });
+        itp('checks correct order after enroll', function () {
+          return setupMultipleFactorEnrollments(
+            factorName, 'ACTIVE', 'OPTIONAL',{'enrolled': 1, 'minimum': 1, 'maximum': 3}
```

Comment:
I think this should be REQUIRED. minimum of 1 means it is REQUIRED. Can you add a test for both REQUIRED and OPTIONAL?